static/js/narrow_banner.js

```diff
@@ -280,6 +292,17 @@ function pick_empty_narrow_banner() {
                     title: $t({defaultMessage: "One or more of these users do not exist!"}),
                 };
             }
+            if (
+                page_params.realm_private_message_policy ===
+                settings_config.private_message_policy_values.disabled.code
+            ) {
```

Comment:
We will want to refactor this when we add other private message policies, but I'm not actually sure what the right interface is for such a function, and this is easy to grep for.